### PR TITLE
OSDOCS#10136: Release note for HashiCorp Vault support for the secret…

### PR DESCRIPTION
**Pull request by bergerhoffer**
…s store operator

Version(s):
4.16

Issue:
https://issues.redhat.com/browse/OSDOCS-10136

Link to docs preview:
https://76719--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes.html#ocp-4-16-storage-secrets-store-csi-driver-vault

QE review:
- [x] QE has approved this change.

Additional information:

The second xref depends on #76657 being merged first to jump to proper section.

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@ropatil010 @dobsonj Can you please review this release note entry?

Preview: https://76719--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes.html#ocp-4-16-storage-secrets-store-csi-driver-vault

**Comment by dobsonj** [comment]
LGTM

File: release_notes/ocp-4-16-release-notes.adoc [hunk 1]
```diff
@@ -157,6 +157,15 @@ For more information about this feature on bare-metal infrastructure, see xref:.
 [id="ocp-4-16-storage"]
 === Storage
 
+[id="ocp-4-16-storage-secrets-store-csi-driver-vault"]
+==== HashiCorp Vault is now available for the {secrets-store-operator}
+
+You can now use the {secrets-store-operator} to mount secrets from HashiCorp Vault. The {secrets-store-operator} is available as a Technology Preview feature.
```
Comment (xenolinux) [inline release_notes/ocp-4-16-release-notes.adoc:163]: Should it be mentioned here that users can use this Operator to mount secrets from HashiCorp Vault -- to a CSI volume in {product-title}.

**Review by xenolinux — COMMENTED** [review]
One query; otherwise LGTM!

/remove-label peer-review-needed
/remove-label peer-review-in-progress
/label peer-review-done

**Comment by bergerhoffer** [comment]
This is ready to merge, but should wait for #76657 to merge first so that we can verify that the xref works properly

**Pull request merged**